src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/ManageController.cs

```diff
@@ -80,7 +86,7 @@ public async Task<IActionResult> Index(IndexViewModel model)
                 var setEmailResult = await this.manageService.SetEmailAsync(user, model.Email).ConfigureAwait(false);
                 if (!setEmailResult.Succeeded)
                 {
-                    throw new ArgumentException($"Wystąpił nieoczekiwany błąd podczas konfigurowania wiadomości e-mail dla użytkownika z identyfikatorem '{user.Id}'.");
+                    throw new InvalidOperationException($"Wystąpił nieoczekiwany błąd podczas konfigurowania wiadomości e-mail dla użytkownika z identyfikatorem '{user.Id}'.");
```

Comment:
add this string to resource "Wystąpił nieoczekiwany błąd podczas konfigurowania wiadomości e-mail dla użytkownika z identyfikatorem {0}" and use String.Format to fill the parameter